content/en/news/releases/1.6.x/announcing-1.6.12/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Announcing Istio 1.6.12
+linktitle: 1.6.12
+subtitle: Patch Release
+description: Istio 1.6.12 patch release.
+publishdate: 2020-10-06
+release: 1.6.12
+aliases:
+    - /news/announcing-1.6.12
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.6.11 and Istio 1.6.12
+
+{{< relnote >}}
+
+## Changes
+
+- **Added** ability to configure domain suffix for multicluster installation  ([Issue #27300](https://github.com/istio/istio/issues/27300))
```

Comment:
2 spaces and spaces in-betwen. Not sure how it changes the format, but the content looks good

Reply:
Removed the extra spac.e